docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -152,6 +152,14 @@ PUT _cluster/settings
   by default, but they can selectively be made optional by setting this setting
   to `true`.
 
+`cluster.remote.${cluster_alias}.transport.ping_schedule`::
+
+  Schedule a regular application-level ping message to ensure that transport
```

Comment:
I think we need to say that this setting sets the time between pings, otherwise it's not clear what values other than `-1` mean. I also think that the "defaults to ... which defaults to ..." in the last sentence might cause confusion. I drafted an alternative:

> Sets the time interval between regular application-level ping messages that are sent to ensure that transport connections to nodes belonging to remote clusters are kept alive. If set to `-1`, application-level ping messages to this remote cluster are not sent. If unset, application-level ping messages are sent according to the global `transport.ping_schedule` setting, which defaults to `-1` meaning that pings are not sent.

I'm not sure this is correct, however. If we set

`transport.ping_schedule: 5s`
`cluster.remote.foo.transport.ping_schedule: -1`

Does this disable pings to the `foo` remote? Should it? I think it'd be useful to be able to do so. I haven't dug into the implementation but there's no test for this case as far as I can see.